zuul.d/jobs.yaml

```diff
@@ -588,3 +589,49 @@
     nodeset: vyos-1.1.8-python38
     vars:
       ansible_test_python: 3.8
+
+# molecule- prefixed jobs match ansible- ones but also enable extra
+# virtualization services that are use by molecule, like docker, podman (TBD)
+# or libvirt (TBD).
+
+- job:
+    <<: [*ansible-tox-molecule, *ansible-tox-linters]
```

Comment:
Hi,

Why don't you use the `parent` key instead to inheritance from the other job? I feel like this kind of advanced YAML incantation don't really improve the readability.

Reply:
Zuul does not supports multiple inheritance and in that case we need to combine properties from 3 sources: 
- python versions/nodesets (each tox pyXY job has a specific nodeset)
- main tox-molecule job which adds the pre-run that installs docker/podman
- custom tox environmnets

As you can see is not merged yet, but apparently merging anchors worked as desired.

The main benefit here is that a setup like this allow use to contro the nodeset from a single place, or how the molecule pre-run yaml from a single location too.

@goneri You are welcomed to experiment with others setups maybe we find a better setup. I agree that these are not very easy to read but they achieve the goal of avoiding configs from diverging, something that happened a log on with triple jobs. I even have small script that dumps the expanded yaml.

Reply:
Indeed lack of visible (implicit) parent is a source of concern here. That is WIP, so I will continue to play with it and until i bring it in an easy to maintain shape.

If i remember well there was a problem with overriding order but now that I know that this setup works, I can save it and experiment with other setups and diff the results, expanded YAML should remain the same. Maybe I can scrap the double dictionary merge.